woocommerce/functions.php

```diff
@@ -36,7 +40,6 @@ function siteorigin_north_woocommerce_enqueue_scripts( ){
 	if( !function_exists('is_woocommerce') ) return;
 
 	if( 'is_woocommerce' ) {
```

Comment:
Oh, it turns out this was my mistake from earlier. This should also be a call to the function `is_woocommerce`.